README.md

```diff
@@ -4,153 +4,144 @@ The singularity at the center of the KDLP infrastructure black hole.
 
 Section 1: Podman Setup 
 --
+ - Unfortunately, this project uses features that have been merged into podman-compose but as of June 2024,
```

Comment:
Even if we weren't updating this README, I'd avoid starting section 1 of the installation guide with "Unfortunately", but for now it's fine